content/community/podcasts.md

```diff
@@ -1,26 +1,25 @@
 ---
 id: podcasts
-title: Podcasts
+title: الصوتيات (podcasts)
 layout: community
 sectionid: community
 permalink: community/podcasts.html
 ---
 
-Podcasts dedicated to React and individual podcast episodes with React discussions.
+مدونات صوتية وحلقات فرديّة مخصصة للحديث عن React.
 
-## Podcasts {#podcasts}
+## الصوتيات {#podcasts}
 
-- [The React Podcast](https://reactpodcast.simplecast.fm/) - The podcast about everything React.js, hosted by [React Training](https://reacttraining.com)
+- [React Podcast](https://reactpodcast.simplecast.fm/) - بودكاست React.js الشامل برعاية [React تدريب](https://reacttraining.com).
```

Comment:
It looks like this now:
![Screenshot from 2019-08-26 02-39-20](https://user-images.githubusercontent.com/9090674/63659706-cc738000-c7aa-11e9-924c-fe40cd6b4335.png)

Shouldn't we make it:
```suggestion
- [React Podcast](https://reactpodcast.simplecast.fm/) - بودكاست React.js الشامل برعاية [تدريب React](https://reacttraining.com).
```